src/index.js

```diff
@@ -333,6 +328,7 @@ function getMain({ options, entry, format }) {
 const shebang = {};
 
 function createConfig(options, entry, format, writeMeta) {
+	const entryFileName = basename(entry, extname(entry));
```

Comment:
This is incorrect. Please use the existing naming setup that we had, it ensures names are properly stripped:

https://github.com/developit/microbundle/blob/9a4e2b2096d3824661738edb142b4658cf3d9d0b/src/index.js#L505

`index.esm.js` as an input needs to become `index`; yours would have it become `index.esm`, so `index.esm.css` would be output. We need to strip indicators towards ES module format, i.e., `cjs`, `es[m]`, etc.